### ISSUE TITLE
Bug: Owner::Account id serializes to object vs string, breaks NEP-171 compatibility

### ISSUE THREAD

**Issue by zcstarr** [Mintbase/mintbase-core#8]
Summary: 
nft_token and the token object doesn't match a subset of the NEP-171 standard. This breaks compatibility, i.e. checking ownership on chain of a mintbase NFT, without special casing mintbase will break with 3rd party integrations. You can't use the near token standards to deserialize the call. 

Motivation: Fixing the serialization, will also fix 3rd party integrations to be in alignment with the rest of the Near ecosystem. Having this be different requires everyone that wants to integrate with Mintbase to special case mintbase.

Simple Reproduction case : 
`call nft_token(token_id)`
The result will be  
```ts 
{
   owner_id: Account: { "someaccount"}
    ...
}
```
The result should be: 
```ts 
{
   owner_id: "someaccount"
}
```

**Comment by tifrel** [comment]
Fixed by #25. Live on testnet, gonna close issue once we have it on mainnet.

**Comment by tifrel** [comment]
Landed in mainnet
